src/main/java/com/twilio/base/bearertoken/Page.java

```diff
@@ -169,28 +169,33 @@ private static <T> Page<T> buildPage(JsonNode root, List<T> results) {
 
     private static <T> Page<T> buildNextGenPage(JsonNode root, List<T> results) {
         JsonNode meta = root.get("meta");
-        Builder<T> builder = new Builder<T>().url(meta.get("url").asText());
+        Builder<T> builder = new Builder<>();
+        if(meta != null && meta.get("url") != null) {
 
-        JsonNode nextPageNode = meta.get("next_page_url");
-        if (!nextPageNode.isNull()) {
-            builder.nextPageUrl(nextPageNode.asText());
-        }
+            builder = builder.url(meta.get("url").asText());
 
-        JsonNode previousPageNode = meta.get("previous_page_url");
-        if (!previousPageNode.isNull()) {
-            builder.previousPageUrl(previousPageNode.asText());
-        }
 
-        JsonNode firstPageNode = meta.get("first_page_url");
-        if (!firstPageNode.isNull()) {
-            builder.firstPageUrl(firstPageNode.asText());
-        }
+            JsonNode nextPageNode = meta.get("next_page_url");
+            if (!nextPageNode.isNull()) {
+                builder.nextPageUrl(nextPageNode.asText());
+            }
 
-        JsonNode pageSizeNode = meta.get("page_size");
-        if (!pageSizeNode.isNull()) {
-            builder.pageSize(pageSizeNode.asInt());
-        } else {
-            builder.pageSize(results.size());
+            JsonNode previousPageNode = meta.get("previous_page_url");
```

Comment:
There are 2 types of pagination that starship supports, Will this work for types of paginations ?